src/FileList/FitsHduList.cc

```diff
@@ -18,11 +18,8 @@ FitsHduList::FitsHduList(const std::string& filename) {
 }
 
 void FitsHduList::GetHduList(std::vector<std::string>& hdu_list, std::string& error) {
-    // Returns list of hdu num and ext name for primary array and image extensions
-    if (IsCompressedFits(_filename)) {
-        error = "Compressed FITS gz/bz format not supported yet.";
-        return;
-    }
+    // Returns list of hdu num and ext name for primary array and image extensions.
+    // Note: DO NOT USE for compressed FITS, open decompresses entire file.
```

Comment:
Should we not continue to disable this function for compressed files, to safeguard against this being ignored in the future?

Reply:
This now throws an exception rather than the previous error message since the error is for the developer not the user.